### ISSUE TITLE
Default service account has no permissions to create PVC

### ISSUE THREAD

**Issue by pat-s** [woodpecker-ci/helm#10]
With NEXT (woodpecker-agent@sha256:6a625b1a1b40f7f840a3a4da7230f45735232fa14b45402bcad10b27a7faf8c3) ) I get on a fresh deployment

```
woodpecker-agent 10:43PM WRN cancel signal received error="rpc error: code = Unknown desc = Step finished with exitcode 1, persistentvolumeclaims is forbidden:  │
│ User \"system:serviceaccount:woodpecker-agent:woodpecker-agent\" cannot create resource \"persistentvolumeclaims\" in API group \"\" in the namespace \"woodpeck │
│ er-agent\""
```

with 

```yml
env:
  WOODPECKER_SERVER: woodpecker-server.woodpecker-server.svc.cluster.local:9000
  WOODPECKER_BACKEND_K8S_NAMESPACE: woodpecker-agent
  WOODPECKER_BACKEND_K8S_STORAGE_CLASS: ebs-sc
  WOODPECKER_BACKEND: kubernetes
  WOODPECKER_BACKEND_K8S_STORAGE_RWX: false
```

I'd expect here that the serviceaccount created would have the required scopes to create the PVC.

**Comment by pat-s** [comment]
Seems like this only applies to 0.15.6 and not to NEXT.

The differences between these too are really confusing and users don't really see if certain fixes/changes are in NEXT or in 0.15.6.

**Comment by Tchoupinax** [comment]
Hey,

Just for next reader, it works only if `WOODPECKER_BACKEND_K8S_NAMESPACE` is the namespace your woodpecker is running in. It makes sense, but I did not do that at the beginning and I was looking why it did not work.
